addon/actions.js

```diff
@@ -107,10 +107,10 @@ function findDefaults (value, path, model, resolveRef) {
 
 export function validate (bunsenId, inputValue, renderModel, validators) {
   return function (dispatch, getState) {
-    const formValue = getState()['value']
+    let formValue = getState().value
     const previousValue = _.get(formValue, bunsenId)
 
-    if (previousValue === undefined) {
+    if (inputValue === undefined && previousValue === undefined) {
```

Comment:
It seems we were opting for a default over the newly set value when one was provided.